esp32-wroom-rp/src/spi.rs

```diff
@@ -149,7 +149,7 @@ where
 
     fn get_socket(&mut self) -> Result<Socket, Error> {
         let operation =
-            Operation::new(NinaCommand::GetSocket, 1).with_no_params(NinaNoParams::new(""));
+            Operation::new(NinaCommand::GetSocket, 1).with_no_params(NinaNoParams::new("").into());
```

Comment:
@calebbourg does the new design change how we do no param commands at all? Perhaps you didn't get a chance to rethink this just yet - I'm just curious.